iotfunctions/db.py

```diff
@@ -1419,7 +1420,7 @@ def read_sql_query(self, sql, parse_dates=None, index_col=None, requested_col_na
         tic = time()
         df = pd.read_sql_query(sql=sql, con=self.connection, **kwargs)
         toc = time()
-        logger.info(f'query execution time: {toc - tic} seconds')
+        logger.info(f"exec_time_secs={toc - tic:.2f}s sql={' '.join(str(sql).split())}")
```

Comment:
is this going to print the entire SQL statement on one line? what was the change (i.e. before it was logging on multiple lines)?

Reply:
yes, this log will print the log as requested in issue1919
the sql logged earlier had new line characters in the sql statement which is why it was printing in multiple lines 
`' '.join(str(sql).split())` statement makes sure we remove these new line characters and any extra whitespaces before printing 
I've also retained the log where the sql printing on multiple line in case someone else was using it